effect3/src/main/scala/org/typelevel/catbird/util/effect/FutureInstances.scala

```diff
@@ -15,7 +15,7 @@ trait FutureInstances {
 
       final override def forceR[A, B](fa: Future[A])(fb: Future[B]): Future[B] =
         fa.liftToTry.flatMap { resultA =>
-          resultA.handle(Monitor.catcher)
+          resultA.handle[Any](Monitor.catcher)
```

Comment:
`Any` was the inferred type, but with sbt-tpolecat, an inferred `Any` ends up as an error because it is often incorrect.